src/components/post/PostBlock.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "../../styles/PostBlock.scss";
+import DropdownButton from "react-bootstrap/DropdownButton";
+import PostDropDown from "./PostDropDown";
+import moreIcon from "../../images/more-icon.svg";
+
+class PostBlock extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
Is this state being used anywhere? If not, we can just remove this constructor.

Reply:
Eslint will complain if I remove this.state as then the constructor would be useless, but we'll have to use state once we have comments added in,  so I think we should just leave it there